### ISSUE TITLE
Text::LocaleDomain 1.32' __find_domain() Errors on Windows

### ISSUE THREAD

**Issue by theory** [gflohr/libintl-perl#7]
Looks like it was broken in 8c2b356, where I guess `abs_path` barfs when `$found_dir` is `undef`. For example, in [this Sqitch test failure](https://github.com/sqitchers/sqitch/runs/3900054856?check_suite_focus=true), it chokes on [this bit of Cwd in Perl 5.16](https://github.com/Perl/perl5/blob/fd04d42d38f4751b981eb6e9213ee1ab8ef11ea6/dist/Cwd/Cwd.pm#L630-L632). Here's the stack trace:

```
# No such file or directory at C:/hostedtoolcache/windows/perl/5.16.3-thr/x64/site/lib/Locale/TextDomain.pm line 415.
#  at C:/hostedtoolcache/windows/perl/5.16.3-thr/x64/lib/Cwd.pm line 354.
# 	Cwd::_croak() called at C:/hostedtoolcache/windows/perl/5.16.3-thr/x64/lib/Cwd.pm line 630
# 	Cwd::fast_abs_path() called at C:/hostedtoolcache/windows/perl/5.16.3-thr/x64/site/lib/Locale/TextDomain.pm line 415
# 	Locale::TextDomain::__find_domain() called at C:/hostedtoolcache/windows/perl/5.16.3-thr/x64/site/lib/Locale/TextDomain.pm line 211
# 	Locale::TextDomain::__x() called at D:\a\sqitch\sqitch\lib/App/Sqitch/Command/add.pm line 224
# 	App::Sqitch::Command::add::configure() called at D:\a\sqitch\sqitch\lib/App/Sqitch/Role/ContextCommand.pm line 49
# 	App::Sqitch::Role::ContextCommand::__ANON__() called at (eval 341) line 1
# 	App::Sqitch::Command::add::__ANON__() called at (eval 343) line 2
# 	App::Sqitch::Command::add::configure() called at ./t/add.t line 144
# 	Test::Exception::throws_ok() called at ./t/add.t line 145
```

**Comment by Montclair** [comment]
I have this issue as well.

**Comment by gflohr** [comment]
Does 3a0ab71 fix the problem?

**Comment by Montclair** [comment]
It does for me, thanks!  Please push to cpan.

**Comment by gflohr** [comment]
Fixed with 3a0ab71. Thanks for testing!
